app/controllers/salesforce/cases_controller.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module Salesforce
+  class CasesController < ::Admin::AdminController
+
+    def sync
+      params.require(:topic_id)
+      topic = Topic.find(params[:topic_id])
+      _case = Case.sync!(topic)
```

Comment:
I think all these `_case` variables are named like this because `case` is reserved in ruby, right? Maybe we should call it `salesforce_case` then or `sf_case` so it's more obvious? At first I just thought it was really weird there was a `_` in front